src/policy/policy.h

```diff
@@ -27,6 +27,9 @@ static const unsigned int MAX_P2SH_SIGOPS = 15;
 static const unsigned int MAX_STANDARD_TX_SIGOPS = MAX_BLOCK_SIGOPS/5;
 /** Default for -maxmempool, maximum megabytes of mempool memory usage */
 static const unsigned int DEFAULT_MAX_MEMPOOL_SIZE = 300;
+/** Default for -blocksizeacceptlimit */
```

Comment:
This is a sensitive option, if you set it below 10 then you will create a permanent chain split (a tightening of the rules is a soft fork, and with minor hash power you can extend the minority chain, with major hash power you orphan others